src/main/java/com/mongodb/kafka/connect/util/ConnectConfigException.java

```diff
@@ -36,4 +39,8 @@ public String getName() {
   public Object getValue() {
     return value;
   }
+
+  public String getOriginalMessage() {
```

Comment:
Just a question, couldn't we just use `getMessage()` instead of creating a new method? Or is that getting modified in the super call?

Reply:
Yeap correct, the inherited error messes with the message, so we have to save it.